### PR TITLE
Added path resolve in webpack config to allow for builds on windows platforms

### LINKED ISSUE #1: Broken paths when attempting to build, either that or README needs more than just npm install
webpack is missing from the initial npm install and when I try to do npm run-script dev or webpack

I receive several errors

ERROR in ./src/reducers/payees.js
Module not found: Error: Cannot resolve 'file' or 'directory' ../../lib/data/payees in C:\www\tbyn/src\reducers
 @ ./src/reducers/payees.js 8:14-46

ERROR in ./src/reducers/transactions.js
Module not found: Error: Cannot resolve 'file' or 'directory' ../../lib/data/payees in C:\www\tbyn/src\reducers
 @ ./src/reducers/transactions.js 16:14-46

ERROR in ./src/reducers/transactions.js
Module not found: Error: Cannot resolve 'file' or 'directory' ../../lib/data/transactions in C:\www\tbyn/src\reducers
 @ ./src/reducers/transactions.js 18:20-58

ERROR in ./src/components/workarea/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/css-loader/index.js in C:\www\tbyn/src\components\workarea
 @ ./src/components/workarea/index.css 4:14-82 13:2-17:4 14:20-88

ERROR in ./src/components/workarea/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/style-loader/addStyles.js in C:\www\tbyn/src\components\workarea
 @ ./src/components/workarea/index.css 7:13-74

ERROR in ./src/components/sidebar/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/css-loader/index.js in C:\www\tbyn/src\components\sidebar
 @ ./src/components/sidebar/index.css 4:14-82 13:2-17:4 14:20-88

ERROR in ./src/components/sidebar/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/style-loader/addStyles.js in C:\www\tbyn/src\components\sidebar
 @ ./src/components/sidebar/index.css 7:13-74

ERROR in ./src/containers/navigation/off-budget-account.js
Module not found: Error: Cannot resolve 'file' or 'directory' ../../../lib/data/accounts in C:\www\tbyn/src\containers\navigation
 @ ./src/containers/navigation/off-budget-account.js 9:16-53

ERROR in ./src/containers/navigation/budget-account.js
Module not found: Error: Cannot resolve 'file' or 'directory' ../../../lib/data/accounts in C:\www\tbyn/src\containers\navigation
 @ ./src/containers/navigation/budget-account.js 9:16-53

ERROR in ./src/containers/navigation/closed-account.js
Module not found: Error: Cannot resolve 'file' or 'directory' ../../../lib/data/accounts in C:\www\tbyn/src\containers\navigation
 @ ./src/containers/navigation/closed-account.js 9:16-53

ERROR in ./src/containers/navigation/account/nav.js
Module not found: Error: Cannot resolve 'file' or 'directory' ../../../../lib/data/transactions in C:\www\tbyn/src\containers\navigation\account
 @ ./src/containers/navigation/account/nav.js 9:20-64

ERROR in ./src/containers/navigation/account/link.js
Module not found: Error: Cannot resolve 'file' or 'directory' ../../../../lib/data/transactions in C:\www\tbyn/src\containers\navigation\account
 @ ./src/containers/navigation/account/link.js 9:20-64
webpack: bundle is now VALID.

just to name a few

### PR DESCRIPTION
**Pull request by stephangrobler**
Added JSHint file

Fixes #1

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+var path = require('path');
```
Comment (epixa) [inline webpack.config.js]: Can you change this to a `const` instead?

File: webpack.config.js [hunk 2]
```diff
@@ -26,6 +28,11 @@ module.exports = {
     path: `${__dirname}/dist`,
   },
   resolve: {
+    modulesDirectories: [
+        path.resolve('node_modules'),
+        path.resolve('src'),
+        path.resolve('lib')
```
Comment (epixa) [inline webpack.config.js]: These lines are indented too far, can you remove two spaces?

**Review by epixa — CHANGES_REQUESTED** [review]
There are just some linting issues on this PR, which I've identified inline.  You can always double check that code adheres to the style rules by running `npm run lint`

I should also get pull requests working with circle-ci so they could be flagged automatically, but that's a separate thing.

**Review by epixa — APPROVED** [review]

**Comment by epixa** [comment]
Thanks for the pull request!

**Comment by emudojo** [comment]
yay, this helps quite a bit, thanks @stephanfx and @epixa

**Pull request merged**